backend/src/event/events.dto.ts

```diff
@@ -41,9 +41,9 @@ export class EventsDto {
   @IsString()
   @ApiModelProperty()
   readonly event_type: string;
-  @IsString()
+  @IsArray()
   @ApiModelProperty()
-  readonly age_group_limit: string;
+  readonly age_group_limit: string[];
```

Comment:
Would it be a biggie to change the field name to plural, now that it can hold more than one value?